src/MaterialEditor.Base/NormalMapManager.cs

```diff
@@ -161,6 +163,25 @@ private static Texture2D MakeTextureReadable(Texture tex, RenderTextureFormat rt
                 RenderTexture.ReleaseTemporary(tmp);
             }
         }
+
+        internal static bool IsUncompressedNormalMap(Texture tex)
+        {
+            Texture2D readableTex = MakeTextureReadable(tex);

```

Comment:
Sorry for cutting in.
Memory leak if readableTex is not released.

Reply:
Thank you! Fixed it now